web/src/modules/centre/components/MonthlyWorksheet.vue

```diff
@@ -167,48 +167,38 @@
 </template>
 <script lang="ts">
 import { mapActions } from "pinia"
+
 import { useCentreStore } from "../store"
+import { formatMoney } from "@/utils"
 
 export default {
   name: "MonthlyWorksheet",
   props: ["month"],
   setup() {},
   data: () => ({}),
-  computed: {},
+  computed: {
+    sections() {
+      return this.month.sections
+    },
+  },
+  watch: {},
   methods: {
     ...mapActions(useCentreStore, ["saveWorksheet", "duplicateAprilEstimates"]),
-
-    formatMoney(amount: any, decimalCount = 2, decimal = ".", thousands = ",") {
-      try {
-        decimalCount = Math.abs(decimalCount)
-        decimalCount = isNaN(decimalCount) ? 2 : decimalCount
-
-        const negativeSign = amount < 0 ? "-" : ""
-
-        const i = parseInt((amount = Math.abs(amount || 0).toFixed(decimalCount))).toString()
-        const j = i.length > 3 ? i.length % 3 : 0
-
-        return (
-          negativeSign +
-          "$" +
-          (j ? i.substr(0, j) + thousands : "") +
-          i.substr(j).replace(/(\d{3})(?=\d)/g, "$1" + thousands) +
-          (decimalCount
-            ? decimal +
-              Math.abs(parseFloat(amount) - parseFloat(i))
-                .toFixed(decimalCount)
-                .slice(2)
-            : "")
-        )
-      } catch (e) {
-        console.log(e)
-      }
-    },
-    changeLine(line: any) {
+    formatMoney,
+    changeLineAndPropagate(line: any, lineIndex: number, sectionIndex: number) {
       line.estChildCount = parseInt(line.estChildCount || 0)
       line.actChildCount = parseInt(line.actChildCount || 0)
       line.estComputedTotal = line.monthlyAmount * line.estChildCount
       line.actComputedTotal = line.monthlyAmount * line.actChildCount
+
+      // Bind section 1 to sections 2 and 3
+      // When you update the values in section 1, it will propagated the values to section 2 and 3
+      if (sectionIndex === 0) {
+        this.sections[1].lines[lineIndex].estChildCount = line.estChildCount
+        this.sections[1].lines[lineIndex].actChildCount = line.actChildCount
+        this.sections[2].lines[lineIndex].estChildCount = line.estChildCount
+        this.sections[2].lines[lineIndex].actChildCount = line.actChildCount
```

Comment:
Need to update the computed totals as well.

Reply:
Fixed in https://github.com/icefoganalytics/elcc-data-management/pull/49/commits/e4eba0b48696d624afbd5d8da5e1d2d92ff82d6d